react-frontend/src/components/ImageCarousel.js

```diff
@@ -20,21 +21,48 @@ function ImageCarousel({ images }) {
   }
 
   function selectImage(index) {
-    setCurrentImage(index);
+    setCurrentImageIndex(index);
+  }
+
+  function nextImage() {
+    if (currentImageIndex < images.length - 1) {
+      setCurrentImageIndex(currentImageIndex + 1);
+    } else {
+      setCurrentImageIndex(0);
+    }
+  }
+
+  function prevImage() {
+    if (currentImageIndex > 0) {
+      setCurrentImageIndex(currentImageIndex - 1);
+    } else {
```

Comment:
Is there a reason that this else doesn't just select index zero?

Reply:
If the current image index is, for example, 3, and you click prevImage button it should go to 2, not 0. So it just decrements it.